includes/class-faqs.php

```diff
@@ -200,4 +203,26 @@ public function get_faqs_markup( $style = '', $filter = 'false', $orderby = '',
 		return ob_get_clean();
 
 	}
+
+	/**
+	 * Plugin action links.
+	 *
+	 * Adds action links to the plugin list table
+	 *
+	 * Fired by `plugin_action_links` filter.
+	 *
+	 * @since 1.3.10
+	 *
+	 * @param array $links An array of plugin action links.
+	 *
+	 * @return array An array of plugin action links.
+	 */
+	public function plugin_action_links( $links ) {
+		$settings_link      = sprintf( '<a href="%1$s">%2$s</a>', admin_url( 'edit.php?post_type=faq&page=quick_and_easy_faqs' ), esc_html__( 'Settings', 'quick-and-easy-faqs' ) );
+		$github_link = sprintf( '<a href="%1$s" target="_blank">%2$s</a>', 'https://github.com/inspirythemes/quick-and-easy-faqs', esc_html__( 'Opt In', 'quick-and-easy-faqs' ) );
```

Comment:
Add Documentation label and link https://github.com/inspirythemes/quick-and-easy-faqs?tab=readme-ov-file#documentation instead of Opt In

Reply:
There is no managed documentation available for this plugin yet. Just some basic details on the WordPress directory plugin detail page. Let me know if you want me to totally remove this link.